relay-server/src/actors/envelopes.rs

```diff
@@ -1853,6 +1853,36 @@ impl EnvelopeProcessor {
         }
     }
 
+    fn light_normalize_event(
+        &self,
+        state: &mut ProcessEnvelopeState,
+    ) -> Result<(), ProcessingError> {
+        let client_ip = state.envelope.meta().client_addr().map(IpAddr::from);
+        let user_agent = state.envelope.meta().user_agent();
+
+        let received_at = Some(state.envelope_context.received_at);
+        let max_secs_in_past = Some(self.config.max_secs_in_past());
+        let max_secs_in_future = Some(self.config.max_secs_in_future());
+
+        let breakdowns_config = state.project_state.config.breakdowns_v2.clone();
+
+        if let Some(event) = state.event.value_mut() {
+            validate_transaction(event).map_err(|e| ProcessingError::ProcessingFailed(e))?;
+        }
+
+        light_normalize_event(
+            &mut state.event,
+            client_ip.as_ref(),
+            user_agent,
+            received_at,
+            max_secs_in_past,
+            max_secs_in_future,
+            breakdowns_config,
+        )
+        .map_err(|e| ProcessingError::ProcessingFailed(e))?;
```

Comment:
We could wrap this call in a `metric!(timer(...))` to measure how long light normalization takes.